### DISCUSSION TITLE
Layouts index下@/ alias 找不到模块

### DISCUSSION THREAD

**Discussion by samous0726** [umijs/umi#11430] [General]
<img width="1698" alt="image" src="https://github.com/umijs/umi/assets/25798093/50672d40-877c-4208-a9c5-4bc4ad5cb70a">

**Comment by fz6m** [comment]
你要确保你有这个文件，并且大小写正确，如有问题，需要给一个最小复现。

**Reply by samous0726** [reply]
页面加载 layouts 渲染了6次，useModel执行了2次，这个正常吗？如下配置方案。另外，layout的初始化config里有antd icon，一开始是ok的 后来报错hooks不在正确的位置使用。然后我把这个config文件挪进src就可以了 这是什么鬼。。
<img width="772" alt="image" src="https://github.com/umijs/umi/assets/25798093/287e87ed-a070-4930-b2a9-ee4b377f9075">
<img width="1701" alt="image" src="https://github.com/umijs/umi/assets/25798093/26c44aa2-0fe8-4b85-b6d0-9c7da131958d">

**Reply by fz6m** [reply]
我不是内部的，可以找蚂蚁内部的 bigfish 对接答疑人问问。

在开源反馈区里，如有问题，需要提供一个最小复现。
